.astylerc

```diff
@@ -0,0 +1,11 @@
+--style=kr 
+--indent=spaces=4 
+--align-pointer=name 
```

Comment:
I dislike this one, but I am willing to comply for consistency. This parameter places the "*" by the variable name rather than the type. e.g: "const char *string"

Reply:
I started using the Qt style because that is the default in Qt Creator, and I quickly liked it. Why are you not using Qt Creator yet?! This is one place where I differed, and I think that came from another C++ convention I was trying to follow. I still prefer to align the asterisk with the identifier in MLT. I am willing to conform to the Qt and MLT standard here and OK with it as-is.

Reply:
> Why are you not using Qt Creator yet?!

Because I am old and stubborn - I have 20 years of muscle memory inertia in Eclipse. I probably need to take the hit and learn something new.